contracts/asset-proxy/contracts/src/ERC20BridgeProxy.sol

```diff
@@ -0,0 +1,128 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-utils/contracts/src/LibBytes.sol";
+import "@0x/contracts-utils/contracts/src/LibSafeMath.sol";
+import "@0x/contracts-utils/contracts/src/Authorizable.sol";
+import "@0x/contracts-erc20/contracts/src/interfaces/IERC20Token.sol";
+import "./interfaces/IAssetProxy.sol";
+import "./interfaces/IERC20Bridge.sol";
+
+
+contract ERC20BridgeProxy is
+    IAssetProxy,
+    Authorizable
+{
+    using LibBytes for bytes;
+    using LibSafeMath for uint256;
+
+    // @dev Result of a successful bridge call.
+    bytes4 constant public BRIDGE_SUCCESS = 0xb5d40d78;
+    // @dev Id of this proxy.
+    //      bytes4(keccak256("ERC20BridgeProxy(address,address,bytes)"))
```

Comment:
This should probably use `ERC20Bridge` rather than `ERC20BridgeProxy` to be consistent with the other ids. We should also add an entry for this to `IAssetData`.